package.json

```diff
@@ -36,6 +36,7 @@
     "eslint-config-next": "13.0.6",
     "next": "13.3.1",
     "next-auth": "4.22.1",
+    "node-fetch": "^3.3.2",
```

Comment:
I don't think that this is required. Please remove this from both, `package.json` as well as `package-lock.json` files.